### PR TITLE
Rename all occurences of "gravatar" to "avatar".

### LINKED ISSUE #252: Remove Gravatar references
We use the `gravatar` name all over the place, which is trademarked.

http://tmsearch.uspto.gov/bin/showfield?f=doc&state=4802:az2tn6.2.1

### PR DESCRIPTION
**Pull request by ralphbean**
Fixes #252.

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/tests/buildsys.py [hunk 1]
```diff
@@ -45,33 +45,34 @@
...
+#Link: http://s390.koji.fedoraproject.org/koji/taskinfo?taskID=1755172
+#
+#error building package (arch s390), mock exited with status 1; see build.log for more information
+#
+#Task 1755173 on fedora2.s390.bos.redhat.com
+#Task Type: buildArch (s390x)
+#Link: http://s390.koji.fedoraproject.org/koji/taskinfo?taskID=1755173
+#
+#error building package (arch s390x), mock exited with status 1; see build.log for more information
+#"""
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/tests/buildsys.py]: Was this meant to be included in this PR?

File: fedmsg_meta_fedora_infrastructure/fasshim.py [hunk 2]
```diff
@@ -67,9 +67,9 @@ def avatar_url_from_email(email, size=64, default='retro', dns=False):
         return "https://seccdn.libravatar.org/avatar/%s?%s" % (hash, query)
 
 
-gravatar_url = avatar_url  # backwards compat
-gravatar_url_from_openid = avatar_url_from_openid
-gravatar_url_from_email = avatar_url_from_email
+avatar_url = avatar_url  # backwards compat
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/fasshim.py]: I think we could drop this line :)

File: fedmsg_meta_fedora_infrastructure/fasshim.py [hunk 3]
```diff
@@ -67,9 +67,9 @@ def avatar_url_from_email(email, size=64, default='retro', dns=False):
...
+avatar_url_from_openid = avatar_url_from_openid
+avatar_url_from_email = avatar_url_from_email
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/fasshim.py]: All three actually :)

File: fedmsg_meta_fedora_infrastructure/tests/buildsys.py (hunk 1 above)
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/tests/buildsys.py]: It was.  That test is permanently failing now :(  See the comment at the top:  ``## Same here.  the state in koji changes which makes this test change often.``

File: fedmsg_meta_fedora_infrastructure/fasshim.py (hunk 3 above)
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/fasshim.py]: Agreed!

File: fedmsg_meta_fedora_infrastructure/tests/buildsys.py (hunk 1 above)
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/tests/buildsys.py]: ok thanks

**Comment by pypingou** [comment]
:+1: for me

**Pull request merged**